nmt.go

```diff
@@ -372,3 +372,15 @@ func (n *NamespacedMerkleTree) computeLeafHashesIfNecessary() {
 type leafRange struct {
 	start, end uint64
 }
+
+// MaxNamespace parses the minimum namespace id from a given hash
+func MinNamespace(hash []byte, size namespace.IDSize) []byte {
+	min := make([]byte, 0, size)
+	return append(min, hash[:size]...)
+}
+
+// MaxNamespace parses the maximum namespace id from a given hash
+func MaxNamespace(hash []byte, size namespace.IDSize) []byte {
+	max := make([]byte, 0, size)
+	return append(max, hash[size:size*2]...)
+}
```

Comment:
we don't have to export these

Reply:
Do you think they will ne used outside of this package? IMO they probably won't. I'd keep them private for now.